Source/AwsCommonRuntimeKit/io/TlsContextOptions.swift

```diff
@@ -2,86 +2,72 @@
 //  SPDX-License-Identifier: Apache-2.0.
 
 import AwsCIo
-//swiftlint:disable trailing_whitespace
 public final class TlsContextOptions {
+    private let allocator: Allocator
     var rawValue: UnsafeMutablePointer<aws_tls_ctx_options>
-    
     public static func isAlpnSupported() -> Bool {
         return aws_tls_is_alpn_available()
     }
-    
+
     public init(defaultClientWithAllocator allocator: Allocator = defaultAllocator) {
-        self.rawValue = allocatePointer()
+        self.allocator = allocator
+        self.rawValue = allocator.allocate(capacity: 1)
         aws_tls_ctx_options_init_default_client(rawValue, allocator.rawValue)
     }
-    
+
     #if os(macOS)
     public init(clientWithMtlsCertificatePath certPath: String,
                 keyPath: String,
                 allocator: Allocator = defaultAllocator) throws {
-        let ptr: UnsafeMutablePointer<aws_byte_cursor> = fromPointer(ptr: keyPath.awsByteCursor)
-        self.rawValue = allocatePointer()
-        if aws_tls_ctx_options_init_client_mtls_pkcs12_from_path(rawValue,
-                                                                 allocator.rawValue,
-                                                                 certPath, ptr) != AWS_OP_SUCCESS {
-            throw CRTError(errorCode: aws_last_error())
-        }
-    }
-    
-    public init(clientWithMtlsCert cert: inout ByteCursor,
-                key: inout ByteCursor,
-                allocator: Allocator = defaultAllocator) throws {
-        self.rawValue = allocatePointer()
-        if aws_tls_ctx_options_init_client_mtls_pkcs12(rawValue,
-                                                       allocator.rawValue,
-                                                       &cert.rawValue,
-                                                       &key.rawValue) != AWS_OP_SUCCESS {
-            throw CRTError(errorCode: aws_last_error())
```

Comment:
Nuke this function as we don't want to expose byte cursor. Customers should call the one which takes cert as a String